docs/software/admin.md

```diff
@@ -877,8 +877,8 @@ This output has two main sections: `qset` and `transitive`. The former describes
 
 Entries to watch for in the `qset` section -- describing the node and its quorum set -- are:
 
-  * `agree` : the number of nodes in the quorum set that agree with this instance.
-  * `delayed` : the nodes that are participating to consensus but seem to be behind.
+  * `agree` : the number of nodes in the quorum set that seem to be up and running as expected. The local node has no reason to believe that this node is `delayed`, `disagree` or `missing`. Note that `agree` has nothing to do with SCP terms such as "accept" or "confirming".
+  * `delayed` : the nodes that are participating to consensus in this round but seem to be behind.
```

Comment:
forgot to update the definition of other fields like "missing" (it says "this consensus round")

Reply:
Updated it!